### PR TITLE
controller layer added for movie and movie session

### PR DESCRIPTION
**Pull request by andromash**

### CODE REVIEW

File: src/main/java/com/hibernate/cinema/controller/AuthenticationController.java [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+    private final AuthenticationService authenticationService;
+
+    @Autowired
+    public AuthenticationController(AuthenticationService authenticationService) {
+        this.authenticationService = authenticationService;
+    }
+
+    @PostMapping("/register")
+    public void signUp(@RequestBody UserRequestDto userRequestDto) {
+        authenticationService.register(userRequestDto.getEmail(), userRequestDto.getPassword());
```
Comment (dariiabutorina) [inline src/main/java/com/hibernate/cinema/controller/AuthenticationController.java:21]: Password check?

**Review by dariiabutorina — APPROVED** [review]

File: src/main/java/com/hibernate/cinema/controller/AuthenticationController.java (hunk 1 above)
Comment (RomanDubovskyi) [reply src/main/java/com/hibernate/cinema/controller/AuthenticationController.java:21]: > Password check?

Will be completed later on

File: src/main/java/com/hibernate/cinema/controller/MovieSessionController.java [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+    }
+
+    @PostMapping
+    public void addMovieSession(@RequestBody MovieSessionRequestDto movieSessionDto) {
+        movieSessionService.add(movieSessionMapper.castDtoToMovieSession(movieSessionDto));
+    }
+
+    @GetMapping("/available")
+    public List<MovieSessionResponseDto> getAvailable(@RequestParam Long movieId,
+                                                      @RequestParam String date) {
```
Comment (RomanDubovskyi) [inline src/main/java/com/hibernate/cinema/controller/MovieSessionController.java]: this is optional, but you can consider following approach 
@RequestParam @DateTimeFormat(pattern = "dd.MM.yyyy") LocalDate date

File: src/main/java/com/hibernate/cinema/model/dto/UserRequestDto.java [hunk 3]
```diff
@@ -0,0 +1,31 @@
+package com.hibernate.cinema.model.dto;
+
+public class UserRequestDto {
+    private String email;
+    private String password;
+    private String passwordRepeated;
```
Comment (RomanDubovskyi) [inline src/main/java/com/hibernate/cinema/model/dto/UserRequestDto.java]: we don't use this field for now, let's add when we need it

**Review by RomanDubovskyi — COMMENTED** [review]
good job, just remove repeatPassword field for now

**Review by KseniiaMakarova — APPROVED** [review]

**Pull request merged**